documentation/source/newsfragments/1457.feature.rst

```diff
@@ -0,0 +1,3 @@
+Add ability to provide a custom entry after ``:`` in a form of ``lib_name_X:A_entry_pointX,lib_name_Y:B_entry_point``.
```

Comment:
```suggestion
Add ability to provide a custom entry point after ``:`` in the format ``lib_name_X:A_entry_point,lib_name_Y:B_entry_point``.
```
in/with form of/format? Native speakers needed :)

Reply:
```suggestion
Add ability to provide custom entry points for libraries in ``GPI_EXTRA``. Entry points can be defined after ``:`` in the format ``lib_name_X:A_entry_point,lib_name_Y:B_entry_point``.
```